llama-index-integrations/llms/llama-index-llms-upstage/llama_index/llms/upstage/base.py

```diff
@@ -207,3 +231,50 @@ def get_num_tokens_from_message(self, messages: Sequence[ChatMessage]) -> int:
                 )
         num_tokens += tokens_suffix
         return num_tokens
+
+    @llm_retry_decorator
+    def _chat(self, messages: Sequence[ChatMessage], **kwargs: Any) -> ChatResponse:
```

Comment:
curious why we subclass _chat, but not any other method like _stream_chat or _achat, etc.

Reply:
I guess we only support parse documents with `.chat()` ?

Reply:
There was a mistake in the llama index interface. Fixed.